examples/bri-3/src/bri/zeroKnowledgeProof/models/zeroKnowledgeProofVerificationInput.ts

```diff
@@ -0,0 +1,17 @@
+import { AutoMap } from '@automapper/classes';
+export class ZeroKnowledgeProofVerificationInput {
```

Comment:
what is this thing for? please add a definition to the proposal document which is accompanying the zkp module work. if not used for anything at the moment but is intended for future work please remove it from the codebase and add it to a github issue explaining the intention and the functionality that should be implemented.

Reply:
This is for future implementation. It was added to accommodate ZKP circuits which would be utilised to verify the state history merkle tree.